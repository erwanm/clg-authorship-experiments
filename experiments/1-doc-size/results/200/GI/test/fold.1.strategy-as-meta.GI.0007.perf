0.552384	0.767200	0.720000
