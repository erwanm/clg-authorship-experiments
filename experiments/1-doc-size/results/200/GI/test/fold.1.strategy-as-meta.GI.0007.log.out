apply-config.sh: applying config '/tmp/200/process//selected-meta-configs/fold.1.strategy-as-meta.GI.0007' to cases '/home/moreaue/dump/expe1//results/200/GI/test/only-cases.tmp.s4Z8e7ei'; results written to '/home/moreaue/dump/expe1//results/200/GI/test/fold.1.strategy-as-meta.GI.0007'
apply-config.sh: done.
